dbus/xdg.c

```diff
@@ -198,6 +198,10 @@ static int handle_notify(sd_bus_message *msg, void *data,
 			}
 			free(notif->desktop_entry);
 			notif->desktop_entry = strdup(desktop_entry);
+		} else if (strcmp(hint, "value") == 0) {
+			int32_t progress = 0;
+			ret = sd_bus_message_read(msg, "v", "i", &progress);
```

Comment:
Need an early return in here in case the read fails, like the other blocks have:
```
if (ret < 0) {
	return ret;
}
```